docs/docs-components/COMPONENT_DATA.js

```diff
@@ -1414,6 +1427,7 @@ const GENERAL_COMPONENT_LIST: $ReadOnlyArray<ListItemType> = [
   {
     svg: <NumberField />,
     name: 'NumberField',
+    aliases: ['Number Input', 'Spin Box'],
```

Comment:
I know, "Spin Box" is weird. It's referring to the ability to increment up and down